scripts/ci-jest

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -e  # Exit on non-zero status
+set -u  # Treat unset variables as an error
+
+# Run test based on what's defined in `package.json`
+npm test
```

Comment:
Why are these run outside of Docker? 
I suspect this line should be `docker-compose exec -T web npm test`. Right @davidflanagan ?

Reply:
@davidflanagan said in the Therapist ADR that Jest tests were 10x slower in Docker than on its local host.

I don't see any reasons why it should run inside the container, considering that Travis installs the npm packages anyway and that the result are identical.

Reply:
No, we want to unit test with the exact version of `node` (and `npm`). 

I think that's different for linting because any outcomes from linting does not affect runtime really. 

"Power users" of kuma dev might want to "cheat" and use their much faster host environment if they can but once they're done, the tests should be run in the correct environment.

Reply:
The node version can be defined in travis.yml.

I don't think the `npm` version is of any importance, the worse that can happen is the build doesn't start at all.

I think it does not bring any added value to run it in Docker.

But ultimately it is your decision, if you would like me to change it, I can

Reply:
Would you mind moving it? Unit tests are very relevant to runtime (unlike linting).

Reply:
Ok.